src/build/utils.js

```diff
@@ -0,0 +1,18 @@
+import fse from 'fs-extra';
+
+export { copyFromTo };
+
+async function copyFromTo(files, srcDir = undefined, targetDir = undefined) {
+  let promises = [];
+  for (let source of files) {
```

Comment:
nit, this can be a one-liner: 

```typescript
return Promise.all(
files.map(source =>  
  fse.copy(source,
         source.replace(`${srcDir}/`, `${targetDir}/`), {
        recursive: true,
        overwrite: true,
        dereference: true,
      })
    )
    )
```